flow/connectors/mysql/cdc.go

```diff
@@ -601,3 +623,113 @@ func QValueFromMysqlRowEvent(mytype byte, qkind qvalue.QValueKind, val any) (qva
 	}
 	return nil, fmt.Errorf("unexpected type %T for mysql type %d", val, mytype)
 }
+
+func (c *MySqlConnector) buildTableSchemaLookup(req *model.PullRecordsRequest[model.RecordItems]) error {
```

Comment:
like in pg, correctness is best if this returns the cached map, & we otherwise avoid direct access to map

Reply:
going for no caching, avoiding computing would be nice but can be done as a follow-up